### PR TITLE
(Bug) Button "Finalize" is active after finalization 

### LINKED ISSUE #975: (Bug) Button "Finalize" is active after finalization
### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)
Ropsten

### If you were able to create it, what is the URL of your crowdsale?
`0x16f21ac37b56997ba556a8b2bd0791fd3bfa236fdc2ba42c1d24c788029bb19e`

### Do you have screenshots showing the problem?
See: https://github.com/poanetwork/token-wizard/pull/973#issuecomment-399588201

### Do you see errors in the dev console? If yes, please include a screenshot
No

If you see errors, please right click on them and "Save as..". Zip saved file and attach it to the Issue.
--
---

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #975 

- The field is_finalized  now takes into account whether it is a mintedcapped or dutch crowdsale
- Added variable to disable the button

See images
Crowdsale with all the tokens sold
issue-975-contribute-enable-all-sell

Allow to finalize crowdsale before finish because all the tokens were sold
issue-975-manage-enable-all-sell

Disable button after finish
issue 998-disable-button-min-cap

### CODE REVIEW

**Comment by dennis00010011b** [comment]
@mariano-aguero 
Button `finalize`  disabled if crowdsale time over and not all tokens were sold
screen shot 2018-07-03 at 10 31 38

**Comment by mariano-aguero** [comment]
@dennis00010011b I made some changes, can you test again the same "case" ? Thank you

File: src/components/manage/index.js [hunk 1]
```diff
@@ -444,8 +444,15 @@ export class Manage extends Component {
     }
     const { start_time, end_time } = await getCrowdsaleStartAndEndTimes(...params).call()
     let crowdsaleInfo = await getCrowdsaleInfo(...params).call()
-    if (crowdsaleInfo && crowdsaleInfo.hasOwnProperty('is_finalized')) {
-      crowdsaleInfo.is_finalized = crowdsaleInfo[4]
+
+    if (isMintedCappedCrowdsale) {
+      if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
```
Comment (vbaranov) [inline src/components/manage/index.js]: @mariano-aguero maybe, we could move `if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {` to outside:
```
if (crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
  if (isMintedCappedCrowdsale) {
    crowdsaleInfo.is_finalized = crowdsaleInfo[3]
  } else if (isDutchAuction) {
    crowdsaleInfo.is_finalized = crowdsaleInfo[4]
  }
}
```
?

File: src/components/manage/index.js [hunk 2]
```diff
@@ -444,7 +444,10 @@ export class Manage extends Component {
     }
     const { start_time, end_time } = await getCrowdsaleStartAndEndTimes(...params).call()
     let crowdsaleInfo = await getCrowdsaleInfo(...params).call()
-    if (crowdsaleInfo && crowdsaleInfo.hasOwnProperty('is_finalized')) {
+
+    if (isMintedCappedCrowdsale && crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
```
Comment (mariano-aguero) [inline src/components/manage/index.js]: @vbaranov Of course, what do you think now?

Comment (vbaranov) [reply src/components/manage/index.js]: @mariano-aguero I think we could merge it now 😃. Thanks.

**Pull request merged**